routes/queries.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../db')
+
+module.exports = {
```

Comment:
If you've heard of the MVC framework, this would be the "Model" side. In the future, moving this file to a different folder will help differentiate it from actual routes. We can make a separate MR for restructuring things to their final places -- this is great for now.